components/lib/multiselect/MultiSelect.js

```diff
@@ -981,11 +981,28 @@ export const MultiSelect = React.memo(
             ZIndexUtils.clear(overlayRef.current);
         });
 
+        const onClearIconKeyDown = (event) => {
+            switch (event.code) {
+                case 'Space':
+                case 'NumpadEnter':
+                case 'Enter':
+                    if (props.inline) {
+                        break;
+                    }
+
+                    updateModel(event, [], []);
```

Comment:
i think you want to add `event.preventDefault()` and `event.stopPropagation()` so this does not bubble up to other components

Reply:
Done @melloware